packages/components/src/placeholder/index.js

```diff
@@ -15,10 +17,18 @@ import Icon from '../icon';
  * @return {Object}       The rendered placeholder.
  */
 function Placeholder( { icon, children, label, instructions, className, notices, preview, isColumnLayout, ...additionalProps } ) {
-	const classes = classnames( 'components-placeholder', className );
+	const [ resizeListener, { width } ] = useResizeAware();
+	const classes = classnames(
+		'components-placeholder',
+		( width >= 320 ? 'size-lg' : '' ),
```

Comment:
should we use `is-large`,  `is-medium`, `is-small` as classNames, I think it's more inline we what we've been using elsewhere.

Reply:
I will make this change, spelling out "large" instead of "lg" is definitely a good call.

However I'm waffling a bit — when used on a Button for example, `isSmall` results in the `is-small` class being applied. However for the placeholder, this is beyond user control — those classes are applied automatically based on dimensions. Am I overthinking it? 

For now, I'm pushing the change and then we can either approve/merge if I am overthinking it, or change it to `size-small` if not.

Reply:
I think it's fine, it doesn't have to be guided by a prop. It's just a convention about how we apply CSS classname modifiers.